src/Widgets/DataLabelWidget/DataLabelWidgetComponent.tsx

```diff
@@ -82,6 +86,10 @@ function Text({ value }: { value: unknown }) {
   return value ? value.toString() : localizeNotAvailable()
 }
 
+function NumberText({ value }: { value: number }) {
+  return new Intl.NumberFormat(getCurrentLanguage()).format(value)
```

Comment:
* [ ] FYI From experience users often prefer `{useGrouping: "min2"}` to keep four-digit numbers as a whole for better readability.

Reply:
This seems to be a fairly recent addition to the standard. At least according to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat Safari only supports it with 16.4 or newer (which was released last year).

Reply:
A workaround for older browsers is `useGrouping: Math.abs(value) >= 10000`.